build-tools/packages/build-infrastructure/package.json

```diff
@@ -1,7 +1,6 @@
 {
 	"name": "@fluid-tools/build-infrastructure",
 	"version": "0.52.0",
-	"private": true,
```

Comment:
Great question. The ultimate goal is to replace all of the release group/package handling code in build-cli commands with the new code from build-infra. But rolling out changes to all the commands at once is just too big a change/task, so the plan is to convert commands one-by-one by creating new versions of the commands that use this new base class. That way we can roll out the changes piecemeal.

Long-term, we may not need a separate package for the infra stuff, but at the moment it's much easier for it to be separate right now, because build-cli and fluid-build need the underlying code. Unfortunately, fluid-build cannot depend on build-cli today because there's already a dep going the other way (that is, build-cli depends on fluid-build).